data/tribes/initialization/empire/units.lua

```diff
@@ -1540,6 +1659,10 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_castle",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire military site: Castle
+            lore = pgettext("empire_building", "‘Ceterum censeo Al’thunran delenda est!’ (Furthermore, I think that Al'thunran must be destroyed!)"),
```

Comment:
I thought it was "esse delendam", not sure?

Reply:
Yes, that's the authentic version. However I don't think any of us could find the correct inflection of Al'thunran matching Carthago -> Carthaginem, so I went for the simplified form also given in credible looking sources.

Reply:
Since -an is not a typical Latin ending and barbarian-style apostrophes inside words are also uncommon in Latin, they'd probably Latinize the name, e.g.
Nominative "Urbs Alt(h)unrana" → Accusative "Alt(h)unranam esse delendam"

Reply:
OK, I'll take your word for it. Is Latin still taught in schools in Germany?